common/lib/capa/capa/inputtypes.py

```diff
@@ -471,7 +471,7 @@ def setup(self):
 
     @classmethod
     def get_attributes(cls):
-        _ = lambda text: text
+        from django.utils.translation import ugettext_noop as _
```

Comment:
Is it possible that in `common` we just don't have django?

@nedbat Renzo asked me about this and I was frank that I didn't know why we had this. I appreciate you weighing in - we documented this very poorly though.